packages/actor-query-operation-orderby-sparqlee/lib/ActorQueryOperationOrderBySparqlee.ts

```diff
@@ -1,7 +1,7 @@
 import { Term } from "rdf-js";
-import { termToString } from "rdf-string";
 import { Algebra } from "sparqlalgebrajs";
 import { AsyncEvaluator, isExpressionError } from 'sparqlee';
+import { orderTypes } from 'sparqlee/dist/lib/util/Ordering';
```

Comment:
I would suggest making this a proper export of sparqlee, so it can be include in the import of the previous line.